extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1235,14 +1236,20 @@ export class AmpA4A extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @typedef {Object} ConsentTuple
+   * @property {?CONSENT_POLICY_STATE} consentState
+   * @property {?string} consentString
+   */
+
   /**
    * Gets the Ad URL to send an XHR Request to.  To be implemented
    * by network.
-   * @param {?CONSENT_POLICY_STATE} unusedConsentState
+   * @param {!ConsentTuple} unusedConsentTuple
```

Comment:
Following up on [this comment](https://github.com/ampproject/amphtml/pull/27780#discussion_r409013050): maybe we should make this optional (allowing `undefined` though, not `null`). This param as a required one led to a lot of changes in the tests.